pontoon/base/static/js/main.js

```diff
@@ -143,6 +143,62 @@ $(function () {
         ga('send', 'event', 'ajax', 'request', settings.url);
     });
 
+    /*
+     * Display Pontoon Add-On Promotion, if:
+     *
+     * - Promotion not dismissed
+     * - Add-On not installed
+     * - Page loaded on Firefox or Chrome (add-on not available for other browsers)
+     */
+    setTimeout(function () {
+        var notDismissed = $('#addon-promotion').length;
+        var notInstalled =
```

Comment:
Personally I would prefer `var installed = window.PontoonAddon && window.PontoonAddon.installed`, which is a little easier to read.

Reply:
Good call. I should probably also `notDismissed` -> `dismissed`.